Assets/Scripts/Photos/PhotoCamera.cs

```diff
@@ -83,13 +105,23 @@ public void Capture()
         RenderTexture.active = myRenderTexture;
 
         byte[] bytes = image.EncodeToPNG();
+        PhotoDTO photo = new PhotoDTO(bytes, defaultPhotoFileFormat);
+
+        // Get any capturable objects identified in the camera raycast
+        Transform transform = Camera.transform;
+        LayerMask mask = LayerMask.GetMask("Capturable");
+
+        if (Physics.Raycast(transform.position, transform.forward, out RaycastHit hit, 20.0f, mask))
```

Comment:
this doesn't work very well so I'm still working on it. Just kinda bumbling my way about google right now lol

Reply:
Yeah you'll wanna graphics raycast to the capturables that are in the cameras view cone, instead of physics raycasting.

I'd probably set up a manager that maintains a list of capturables.

Reply:
Physics raycast will ignore everything without a collider iirc